index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="stylesheet" href="style.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="bar-nav">
+        <h1>My Logo</h1>
+        <img src="image/Union.png" alt="Menu icon">
+    </div>
+    <div class="container">
+      <section class="detail one">
+        I'm Mariane<br>Glad to see<br>you
+      </section>
```

Comment:
- [ ] For more semantic reason kindly use `h1` tag instead of `section` to wrap your title